map_node:
  ros__parameters:
    url: 'http://127.0.0.1:80/wms'
    layers: ['imagery']
    dem_layers: ['osm-buildings']
    # srs: 'EPSG:4326'  # default value, disabled for testing purposes
    image_format: 'image/jpeg'
    map_update_overlay_threshold: 0.85
    max_map_radius: 1000
